{{cookiecutter.repo_name}}/.env

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
@wismer can you delete the `.env` file? You might need to update the init script to `mv env.example .env`